packages/react/src/HeroImage/HeroImage.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef, HTMLAttributes, ImgHTMLAttributes, PropsWithChildren } from 'react'
+
+export interface HeroImageProps extends PropsWithChildren<HTMLAttributes<HTMLElement>> {
+  /** The source of the image. */
+  src: ImgHTMLAttributes<HTMLImageElement>['src']
+}
+
+export const HeroImage = forwardRef<HTMLDivElement, HeroImageProps>(
+  ({ children, className, src, ...restProps }: HeroImageProps, ref) => (
+    <div {...restProps} ref={ref} className={clsx('amsterdam-hero-image', className)}>
+      <img alt="" className="amsterdam-hero-image__image" src={src} />
```

Comment:
Perhaps using the Image component is better here and I think the alt text should describe something to screenreaders. Even if it's decorative. But perhaps @alimpens has different thoughts.

This large image could also benefit from being responsive.

Reply:
```suggestion
      <div className="amsterdam-hero-image__wrapper">
        <img alt="" className="amsterdam-hero-image__image" src={src} />
      </div>
```

Reply:
It doesn't use a `srcset`, in stories you're loading a 1280px wide image in a 320px breakpoint. This becomes a performance penalty.

Reply:
People can use `srcset` if they want. Responsive images wasn’t really in scope in my mind, but we could include it here as this is the widest an image will get on our sites.